core/src/test/java/com/klaytn/caver/feature/KIP17Test.java

```diff
@@ -0,0 +1,1070 @@
+package com.klaytn.caver.feature;
+
+import com.klaytn.caver.Caver;
+import com.klaytn.caver.crypto.KlayCredentials;
+import com.klaytn.caver.methods.response.KlayTransactionReceipt;
+import com.klaytn.caver.tx.gas.DefaultGasProvider;
+import com.klaytn.caver.tx.kct.KIP17;
+import com.klaytn.caver.tx.kct.KIP7;
+import com.klaytn.caver.tx.manager.TransactionManager;
+import com.klaytn.caver.utils.ChainId;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.web3j.tx.gas.StaticGasProvider;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+import java.util.List;
+
+import static com.klaytn.caver.base.Accounts.*;
+import static com.klaytn.caver.base.LocalValues.LOCAL_CHAIN_ID;
+import static org.junit.Assert.*;
+
+public class KIP17Test {
+    private static final String sContractName = "NFTTest";
+    private static final String sContractSymbol = "NFT";
+    private static final String sTokenURI = "https://game.example/item-id-8u5h2m.json";
+    private static final String sSTATUS_SUCCESS = "0x1";
+    private static BigInteger sTotalSupply = BigInteger.ZERO;
+    private String sZeroAddr = "0x0000000000000000000000000000000000000000";
+
+    static Caver mCaver;
+    static KlayCredentials mDeployerCredential, mTestCredential, mTestCredential2;
+    static TransactionManager mDeployerTxManager, mTesterTxManger, mTesterTxManger2;
+    static String mContractAddress;
+
+
+    @BeforeClass
+    public static void preSetup() throws Exception {
+        mCaver = Caver.build(Caver.DEFAULT_URL);
+
+        mDeployerCredential = LUMAN;
+        mDeployerTxManager = new TransactionManager.Builder(mCaver, LUMAN).setChaindId(LOCAL_CHAIN_ID).build();
+
+        mTestCredential = BRANDON;
+        mTesterTxManger = new TransactionManager.Builder(mCaver, BRANDON).setChaindId(LOCAL_CHAIN_ID).build();
+
+        mTestCredential2 = WAYNE;
+        mTesterTxManger2 = new TransactionManager.Builder(mCaver, WAYNE).setChaindId(LOCAL_CHAIN_ID).build();
+
+        deployKIP17Contract();
+    }
+
+    //KCT-032
+    public static void deployKIP17Contract() {
+        try {
+            KIP17 token = KIP17.deploy(
+                    mCaver,
+                    mDeployerTxManager,
+                    new StaticGasProvider(DefaultGasProvider.GAS_PRICE, BigInteger.valueOf(6_000_000)),
+                    sContractName,
+                    sContractSymbol
+            ).send();
+
+            mContractAddress = token.getContractAddress();
+            System.out.println("Contract Address is " + mContractAddress);
```

Comment:
```suggestion
            System.out.println("Contract Address is " + mContractAddress);
```
How about checking mContractAddress is defined instead of printing? (KIP7Test also !)

Reply:
Okay. I modified code.
Please see KCT-032 in test code.